hcs/swagger.json

```diff
@@ -438,6 +438,10 @@
         "consulCaFile": {
           "type": "string",
           "title": "consul_ca_file is a base64-encoded ca chain file in PEM format that\nshould be used by the client agent to verify the authenticity of the\nserver"
+        },
+        "consulConnect": {
```

Comment:
Not sure if this ship has sailed, but could this variable be named `enableConsulConnect` or `consulConnectEnabled`?

Reply:
Seems reasonable to me. I'll make the change

Reply:
Spoke with @joeandaverde and @jipperinbham. We're keeping this as-is. Azure doesn't support booleans at the moment.